Source/Dafny/Dafny.atg

```diff
@@ -386,7 +386,8 @@ bool IsSetDisplay() {
   if (la.kind == _lbrace) return true;
   int k = scanner.Peek().kind;
   if (la.kind == _iset && k == _lbrace) return true;
-  if (la.kind == _multiset && (k == _lbrace || k == _openparen)) return true;
+  // Until there is a multiset comprehension, there is no point in checking the next item.
```

Comment:
I would remove the `//` and the `/*` comment. I don't think a future implementor of multiset comprehensions needs these.